frappe/utils/install.py

```diff
@@ -18,9 +18,6 @@ def before_install():
 
 
 def after_install():
-	# reset installed apps for re-install
-	frappe.db.set_global("installed_apps", '["frappe"]')
-
```

Comment:
this is not needed here as we update `installed_apps` default key before running the `after_install` hook in the `install_apps` function 

https://github.com/frappe/frappe/blob/8a1889301a85d74fa88f9fbb8f6f8d4e42069b6b/frappe/installer.py#L273-L281